projects/storefrontapp-e2e-cypress/cypress/e2e/regression/user_access/recaptchaV2.core-e2e.cy.ts

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
It could be nice to stress test our captcha feature in e2e tests so that we can make sure captcha will not fail if we do things like (for example, but does not necessarily need to be these scenarios):
- Register multiple users
- Navigate to and from registration and try register
- Register incorrectly before correctly
In essence, we want to make sure that we are not reinitializing our script or doing anything funny if we were to trigger our captcha service multiple times.

Reply:
> register multiple users
why do captcha need this? The original test for register component only register one user.
> Navigate to and from registration and try register
this scenario has been covered. 
> Register incorrectly before correctly
I will add this later.